src/spec/PointerInputSpec.ts

```diff
@@ -0,0 +1,111 @@
+/// <reference path="jasmine.d.ts" />
+
+/// <reference path="TestUtils.ts" />
+
+function executeMouseEvent(type: string, button: ex.Input.PointerButton, x: number, y: number, target: HTMLElement) {
```

Comment:
I am leaving the "target" as an optional property so I can add the Wheel Event tests once PR #809 is merged. In order to test I will need to trigger wheel events on both the canvas and document.